### DISCUSSION TITLE
🌊 Making Waves 🌊

### DISCUSSION THREAD

**Discussion by LivingWithHippos** [LivingWithHippos/unchained-android#181] [Feedback]
### 🏷️ New features:

- [x] multi select in folder screen
- [x] copy and share all in folder screen
- [x] paste multiple links in new download screen
- [x] choose the start-up screen from settings

### 🔎  Enhancements:

- [x] show popup to choose sorting
- [x] new theme (Waves 1)

### 🦟 Bug fixes:

- [x] clear results on new search
- [x] DNS over HTTPS is now working
- [x] should crash less

#### Notes:

- as usual there are two APKs, a release and a debug one (the bigger one with "debug" in the name), the debug version will allow me to fix bugs more easily but if your app does not crash you don' t need to use it
- both releases on GitHub have all the search plugins pre-installed
- it will be uploaded on Google Play a little later after getting feedback from the people using the GitHub version

<hr /><em>This discussion was created from the release <a href='https://github.com/LivingWithHippos/unchained-android/releases/tag/4.27.2-beta'>🌊 Making Waves 🌊</a>.</em>

**Comment by Invictaz** [comment]
Amazing release again!
